src/components/common/ClientIntakeForm/IntakeForms/Finances.js

```diff
@@ -1,175 +1,189 @@
-                    <label>Medical Bills</label>
-                  </Checkbox>
-                </Form.Item>
-                <div className="Finances_Form_Amount_Container">
-                  <Form.Item
-                    label="Amount::"
-                    className="Finances_Form_Amount_Label"
-                  >
-                    <Input
-                      placeholder="$0"
-                      name="Amount"
-                      className="Finances_Form_Amount_Input"
-                    />
-                  </Form.Item>
-                </div>
-              </Form.Item>
+              ))}
+              <Input
+                placeholder="$0"
+                name="Amount"
+                className="Finances_Form_Amount_Input"
+                style={{ width: '30%' }}
+              />
+              ;
             </div>
           </section>
 
-          <section>
-            <div className="Finances_Form_Container_Styles">
-              <Form.Item className="Finances_Form_Container_Styles_Subsection2">
-                <Form.Item>
-                  <Checkbox value="creditCard">
-                    <label>Credit Card</label>
-                  </Checkbox>
-                </Form.Item>
-                <div className="Finances_Form_Amount_Container">
-                  <Form.Item
-                    label="Amount::"
-                    className="Finances_Form_Amount_Label"
-                  >
-                    <Input placeholder="$0" name="Amount" />
-                  </Form.Item>
-                </div>
-
-                <Form.Item>
-                  <Checkbox value="autoLoan">
-                    <label>Auto Loan</label>
-                  </Checkbox>
-                </Form.Item>
-                <div className="Finances_Form_Amount_Container">
-                  <Form.Item
-                    label="Amount::"
-                    className="Finances_Form_Amount_Label"
-                  >
+          {/*  TYPES OF DEBT */}
+          <section style={{ width: '45%' }}>
+            <div className="Finances_Form_Label">Types of Debt:</div>
+            <div
+              className="Finances_Form_Container_Styles"
+              style={{ display: 'flex', flexDirection: 'column' }}
+            >
+              {typesOfDebt.map(info => (
+                <div key={info.name} className="Finances_Form_Container_Styles">
+                  <Form.Item>
+                    <Checkbox
+                      name={info.name}
+                      checked={formValues[info.name]}
+                      value={formValues[info.name]}
+                      onChange={onChange}
+                    >
+                      <label>{info.label}</label>
+                    </Checkbox>
                     <Input
                       placeholder="$0"
                       name="Amount"
                       className="Finances_Form_Amount_Input"
+                      style={{ width: '55%', padding: 1 }}
```

Comment:
Did you write inline styles for temporary use?

Reply:
yes, with the intent that it will eventually be styled using less